mingw-w64-git-credential-manager-core/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer: Johannes Schindelin/Matthew J Cheetham
+
+_realname="git-credential-manager-core"
+pkgbase="mingw-w64-${_realname}"
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=2.0.164.25618
+pkgrel=1
```

Comment:
Would you mind setting this to `0` in the beginning? The convention in Git for Windows' SDK is to increase the `pkgrel` (or increment `pkgver` and reset `pkgrel`) as part of the build that deploys the package.

While at it, could I ask you to [separate this new package definition out](https://git-scm.com/docs/git-rebase#_splitting_commits) into its own commit?

Reply:
Done and done!